README.md

```diff
@@ -141,6 +141,24 @@ set of tests. We're not quite there yet, but [we're getting close][codecov].
 [codecov]: https://codecov.io/github/WordPress/Requests/
 
 
+Requests and PSR-7/PSR-18
+-------------------------
+
+[PSR-7][psr-7] describes common interfaces for representing HTTP messages.
+[PSR-18][psr-18] describes a common interface for sending HTTP requests and receiving HTTP responses.
+
+Both PSR-7 as well as PSR-18 were created after Requests' conception.
+At this time, there is no intention to add a native PST-7/PSR-18 implementation to the Requests library.
+
+However, the amazing [Artur Weigandt][art4] has created a [package][requests-psr-18], which allows you to use Requests as a PSR-7 compatible PSR-18 HTTP Client.
```

Comment:
```suggestion
However, [Artur Weigandt][art4] has created a [package][requests-psr-18], which allows you to use Requests as a PSR-7 compatible PSR-18 HTTP Client.
```

Reply:
@Art4 Are you sure ? I wrote it cause I mean it. You've done an amazing job creating that package, so I'm happy for us to warmly endorse it.

Reply:
@jrfnl Sorry, I'm very bad at getting compliments. Please continue as you like.